lightning/src/ln/channelmanager.rs

```diff
@@ -9124,22 +9124,19 @@ macro_rules! create_offer_builder { ($self: ident, $builder: ty) => {
 		let context = MessageContext::Offers (
 			OffersContext::InvoiceRequest { nonce }
 		);
+
+		let offer_builder = OfferBuilder::deriving_signing_pubkey(node_id, expanded_key, nonce, secp_ctx)
```

Comment:
Just call this `builder`. Any methods on it consume it so it's ok to reuse the name later. Same for `refund_builder`.

Reply:
Done! [pr3246.11](https://github.com/shaavan/rust-lightning/commits/pr3246.11)
Thanks!